app/templates/gentelella/guest/event/cfs.html

```diff
@@ -100,7 +100,7 @@ <h4 class="modal-title">{{ _("Add Speaker Information") }} </h4>
                                                 <h4>{{ _("Existing Sessions") }} </h4>
                                                 <p>No Existing Sessions! Please Click on <b>Add Session Proposal</b> to submit a new session.<br><br></p>
                                             </div>
-                                            <a href="{{ url_for('my_sessions.process_speaker_view',speaker=user_speaker[0]['id']) }}" type="button" class="btn btn-info" id = "speaker-button">{{ _("Edit Speaker Details") }}</a>
+                                            <a href="{{ url_for('my_sessions.process_speaker_view',speaker_id=user_speaker[0]['id']) }}" type="button" class="btn btn-info" id = "speaker-button">{{ _("Edit Speaker Details") }}</a>
```

Comment:
@niranjan94 

I don't know how to remove it , as I squashed the commits it might have gone into .
But how do I remove it now ?
Any help would be great ( I have googled some things but I don't know what exactly am I looking to remove so can't get good results from google ).